pyeudiw/satosa/backend.py

```diff
@@ -439,6 +443,7 @@ def request_endpoint(self, context, *args):
                 attestation=context.http_headers['HTTP_AUTHORIZATION']
             )
         except Exception as e:
+            self._log(context, level='error', message=str(e))
```

Comment:
str(e) is not enough, please envelope it in a format string and give more context about the nature of the exception

Reply:
Resolved in [ec8559a](https://github.com/italia/eudi-wallet-it-python/pull/89/commits/ec8559aa058253962e9d78bae647587522324e63)